docs/user-guide/network-configuration/cluster-networking.md

```diff
@@ -0,0 +1,76 @@
+
+| Stack      | SDN | OVN |
+|------------|-----|-----|
+| IPv4       | Yes | Yes |
+| IPv6       | No  | Yes |
+| Dual-stack | No  | Yes |
+
+## Limitations
+
+### SDN
+
+* With single node OpenShift (SNO), the SDN controller is not supported.
+* The SDN controller does not support IPv6.
+
+### OVN-Kubernetes
+
+https://docs.openshift.com/container-platform/4.10/networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.html#nw-ovn-kubernetes-limitations_about-ovn-kubernetes
+
+## Cluster Network
+
+Cluster network is a network from which every pod deployed in the cluster gets its IP address. Given that the workload may live across multiple nodes forming the cluster, it's important for the network provider to be able to easily find an individual node based on the pod's IP address. In order to do this, `clusterNetwork.cidr` is further split into multiple subnets of the size defined in `clusterNetwork.hostPrefix`.
+
+Host prefix specifies a lenght of the subnet assigned to each individual node in the cluster. An example of how a cluster may assign addresses for the multi-node cluster
+
+```yaml
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+```
+
+Creating a 3-node cluster using the snipet above may create the following network topology
+
+* pods scheduled in the node #1 get IPs from `10.128.0.0/23`
+* pods scheduled in the node #2 get IPs from `10.128.2.0/23`
+* pods scheduled in the node #3 get IPs from `10.128.4.0/23`
+
+Explaining OVN-K8s internals is out of scope of this document, but the pattern described above gives us an easy way to route pod-to-pod traffic between different nodes without keeping a big list of mapping between pods and their corresponding nodes.
+
+## Machine Network
+
+Machine network is a network used by all the hosts forming the cluster to communicate with each other. This is also the subnet that has to include API and Ingress VIPs.
+
+## SNO vs Multi-Node Cluster
+
+Depending on whether a Single Node OpenShift or a Multi-Node cluster is deployed, different values are mandatory. The table below explains this in more detail.
+
+| Parameter      | SNO       | Multi-Node Cluster     |
+|----------------|-----------|------------------------|
+| clusterNetwork | Required  | Required               |
+| serviceNetwork | Required  | Required               |
+| machineNetwork | Required  | Forbidden in DHCP mode |
+| apiVIP         | Forbidden | Required               |
+| ingressVIP     | Forbidden | Required               |
+
+## Airgapped environments
+
+The workflow for deploying a cluster without Internet access has some prerequisites which are out of scope of this document. You may consult the [Zero Touch Provisioning in the hard way git repo](https://github.com/jparrill/ztp-the-hard-way/blob/main/docs/disconnected-ZTP-flow-hub-deployment.md) for some insights.
```

Comment:
```suggestion
The workflow for deploying a cluster without Internet access has some prerequisites which are out of scope of this document. You may consult the [Zero Touch Provisioning the hard way git repo](https://github.com/jparrill/ztp-the-hard-way/blob/main/docs/disconnected-ZTP-flow-hub-deployment.md) for some insights.
```

Reply:
Fixed, but it's the name in the upstream repo, just copy-paste